sn_api/src/app/keys.rs

```diff
@@ -102,9 +102,7 @@ mod tests {
         assert_eq!(url, url2);
 
         match keypair {
-            Keypair::Bls(_) => {
-                return Ok(());
-            }
+            Keypair::Bls(_) => Ok(()),
             _ => {
                 return Err(eyre!("A BLS keypair should be generated by default."));
             }
```

Comment:
Apply the same change to the err arm as well?
```suggestion
            _ => Err(eyre!("A BLS keypair should be generated by default."))
```

Reply:
Ah, thanks. Clippy only flagged the `Ok` one. Will change this.